### DISCUSSION TITLE
Number of clients limit for a single stream

### DISCUSSION THREAD

**Discussion by yateya** [bluenviron/mediamtx#2496] [Questions]
### Question

Hi,

I am using mediamtx to simulate RTSP camera for testing a rtsp client application. After ~20 client connections, client connections get disconnected randomly. 

Is there any documentation about scaling streams to big number of clients?

mediamtx is running alone on a linux machine, intel core i3 10gen with 16G Ram.
Video to play is H264, resolution 2560x1440, 20 fps and bitrate 5mbps.

**Comment by aler9** [comment]
Hello, the bottleneck is almost always the throughput of your network, while CPU and RAM are not influent. Make sure that any router, cable, wireless link, internet provider between the server and your clients can handle the desired bandwidth.

**Reply by aler9** [reply]
if you want to test bandwidth, i usually use iperf:
https://www.dell.com/support/kbdoc/en-us/000139427/how-to-test-available-network-bandwidth-using-iperf

**Comment by curtishall** [comment]
@yateya Did you resolve this?  I'm testing on a dummy network with 30GB bandwidth between nodes (according to iperf).  I'm testing with 250 RTSP connections

**Reply by aler9** [reply]
There is no way to tell whether your problem is related to the one described in this discussion without further informations. If you still need support, open a discussion or a bug report and provide details about your setup and issue.
